examples/torch/pearl_ml1_push.py

```diff
@@ -0,0 +1,163 @@
+"""PEARL ML1 example."""
+
+import akro
+from metaworld.benchmarks import ML1
+import numpy as np
+
+from garage.envs import normalize
+from garage.envs.base import GarageEnv
+from garage.envs.env_spec import EnvSpec
+from garage.experiment import LocalRunner, run_experiment
+from garage.experiment.meta_evaluator import MetaEvaluator
+from garage.experiment.task_sampler import SetTaskSampler
+from garage.sampler import LocalSampler
+from garage.torch.algos import PEARL
+from garage.torch.algos.pearl import PEARLWorker
+from garage.torch.embeddings import MLPEncoder
+from garage.torch.policies import ContextConditionedPolicy
+from garage.torch.policies import TanhGaussianMLPPolicy
+from garage.torch.q_functions import ContinuousMLPQFunction
+import garage.torch.utils as tu
+
+params = dict(
+    num_epochs=1000,
+    num_train_tasks=50,
+    num_test_tasks=10,
+    latent_size=7,
+    net_size=300,
+    meta_batch_size=16,
+    num_steps_per_epoch=4000,
+    num_initial_steps=4000,
+    num_tasks_sample=15,
+    num_steps_prior=750,
+    num_extra_rl_steps_posterior=750,
+    num_steps_per_eval=450,
+    batch_size=256,
+    embedding_batch_size=64,
+    embedding_mini_batch_size=64,
+    max_path_length=150,
+    reward_scale=10.,
+    use_information_bottleneck=True,
+    use_next_obs_in_context=False,
+    use_gpu=True,
+)
+
+
+def run_task(snapshot_config, *_):
```

Comment:
please use the `wrap_experiment` interface for this instead. that means each entry in `params` will become an argument to your experiment function here.